### PR TITLE
Add linters and pre-commit hook

### PR DESCRIPTION
**Pull request by csuriel**
# Add liters

Trello: [trello card link](https://trello.com/c/0uXwr8d3/8-add-pre-commit-and-pre-push-hooks)

## Description

- Added some tools that checks the code format before commits

### Checklist:

- [x] My branch name is descriptive and has the correct format (e.g., name-of-the-branch)
- [x] I have added a proper title.
- [x] I have tested my code locally.
- [x] I have self-review my code to make sure it follows the style [guidelines of this project](../README.md)
- [x] I have assigned a code reviewer.
- [x] I have updated the Trello ticket with the PR link.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,17 @@ Projects are created for each topic in [Freecodecamp](https://www.freecodecamp.o
 
 Visit [trello board](https://trello.com/b/XGKSW6rb/nathaly-learning-path) for project status and progress.
 
+## Prerequisites
+
+- [NodeJS](https://nodejs.org/en/download).
```
Comment (csuriel) [inline README.md:10]: We need to ensure these extensions are installed in VS Code so that the linters will work correctly.

File: cat.html [hunk 2]
```diff
@@ -1,72 +1,91 @@
-<!DOCTYPE html>
+<!doctype html>
```
Comment (csuriel) [inline cat.html:1]: formatted this file to match linter requirements (mostly indentation changes)

File: index.html [hunk 3]
```diff
@@ -1,29 +1,30 @@
-<!DOCTYPE html>
+<!doctype html>
```
Comment (csuriel) [inline index.html:1]: formatted this file to match linter requirements (mostly indentation changes)

File: stylesss.css [hunk 4]
```diff
@@ -3,10 +3,11 @@ h1 {
 }
```
Comment (csuriel) [inline stylesss.css:3]: Some notations were changed as the Stylelint requirements. [see here](https://stylelint.io/user-guide/rules/color-function-notation/) for more details.

**Review by vnathaly — APPROVED** [review]

**Pull request merged**